test/integration/OrderBook.spec.ts

```diff
@@ -62,7 +67,7 @@ describe('OrderBook', () => {
       }
       return client;
     };
-    orderBook = new OrderBook(loggers.orderbook, db.models, false, pool, swaps);
+    orderBook = new OrderBook(loggers.orderbook, db.models, limits, false, pool, swaps);
```

Comment:
Rather than pass the limits around in constructors (which already are taking lots of parameters) I think we can put the limits in the `constants` folder and `import` them where they are needed.